### PR TITLE
Add new speech bubble variant to bubble pattern (for hypothesis)

### PR DESCRIPTION
**Pull request by davidcmoulton**

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
When we add this should JS only?

File: assets/sass/patterns/atoms/buttons.scss [hunk 1]
```diff
@@ -85,5 +85,55 @@
...
+    @include colors($color-primary);
+    display: block;
+    font-family: $font-primary;
+    @include font-size-and-vertical-height(48, 36);
+    @include height(36);
+    @include padding(12 0 0);
+    position: relative;
+    @include width(42);
+
+    &:after {
```
Comment (thewilkybarkid) [inline assets/sass/patterns/atoms/buttons.scss]: Is it possible to make this clickable too?

Comment (davidcmoulton) [reply assets/sass/patterns/atoms/buttons.scss]: I've played with some hacky approaches and the answer is yes in theory, but want to find a nicer approach to it.

**Comment by davidcmoulton** [comment]
@thewilkybarkid for this useage of it, js is required (getting to that), but it's conceived as a new button variant, so shouldn't be constrained to be js-only per se.

**Pull request merged**